bin/node/runtime/src/lib.rs

```diff
@@ -269,7 +268,6 @@ impl InstanceFilter<Call> for ProxyType {
 			),
 			ProxyType::Governance => matches!(
 				c,
-				Call::Democracy(..) |
 				Call::Council(..) |
```

Comment:
We can probably take out council and technical commitee as well

Reply:
Sounds good. I can do that in a follow-on PR to this one

Reply:
Actually I noted this in the previous PR - `committee` is tied pretty directly to a lot of things especially when you start running the tests. I'll do this next.

Reply:
ah ok we can leave it then